src/ROOTWriter.cc

```diff
@@ -15,8 +15,14 @@ namespace podio {
     m_store(store),
     m_file(new TFile(filename.c_str(),"RECREATE","data file")),
     m_datatree(new TTree("events","Events tree")),
-    m_metadatatree(new TTree("metadata", "Metadata tree"))
-  {}
+    m_metadatatree(new TTree("metadata", "Metadata tree")),
```

Comment:
Created issue #93 to discuss/review the implicit attachment of the created TTrees to the data file.